canonical-websites/www.ubuntu.com
# Split the dynamic contact functionality out

## Done
Split out the contact us functionality into its own files for reuse.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to /core click a contact link/button
- Check the contact us modal loads without calling the JS function
- Check the same for contact links in the iot section

Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4597
